Modules/Microsoft365DSC/DSCResources/MSFT_AADConnectorGroupApplicationProxy/MSFT_AADConnectorGroupApplicationProxy.schema.mof

```diff
@@ -0,0 +1,16 @@
+
+[ClassVersion("1.0.0.0"), FriendlyName("AADConnectorGroupApplicationProxy")]
+class MSFT_AADConnectorGroupApplicationProxy : OMI_BaseResource
+{
+    [Write, Description("The name associated with the connectorGroup.")] String Name;
+    [Write, Description("The region the connectorGroup is assigned to and will optimize traffic for. This region can only be set if no connectors or applications are assigned to the connectorGroup. The possible values are: nam (for North America), eur (for Europe), aus (for Australia), asia (for Asia), ind (for India), and unknownFutureValue."), ValueMap{"nam","eur","aus","asia","ind","unknownFutureValue"}, Values{"nam","eur","aus","asia","ind","unknownFutureValue"}] String Region;
+    [Key, Description("The unique identifier for an entity. Read-only.")] String Id;
```

Comment:
Again, we should probably make Name the Key property to ensure we can replicate across tenants.